src/pyOpenMS/pxds/MSSpectrum.pxd

```diff
@@ -135,4 +135,17 @@ cdef extern from "<OpenMS/KERNEL/MSSpectrum.h>" namespace "OpenMS":
         void setIntegerDataArrays(libcpp_vector[IntegerDataArray] ida) except + nogil  # wrap-doc:Sets the additional int data arrays to store e.g. meta data
         void setStringDataArrays(libcpp_vector[StringDataArray] sda) except + nogil  # wrap-doc:Sets the additional string data arrays to store e.g. meta data
 
+        
+
+# Unique identifier for the spectrum (e.g., from the mzML file)
+        String getNativeID() except + nogil  # wrap-doc:Returns the spectrum's native ID (e.g., "controllerType=0 controllerNumber=1 scan=100")
```

Comment:
are those needed? Shouldn't they be wrapped through the SpectrumSettings class it derives from?
See https://github.com/OpenMS/OpenMS/blob/b36cdec7d0ab3d89162b6d9477ed6b1b79362152/src/pyOpenMS/pxds/SpectrumSettings.pxd#L25